docs/source/documentation.rst

```diff
@@ -76,17 +88,8 @@ Then, activate your virtual environment::
 
        conda deactivate
 
-AxonDeepSeg
------------
-.. WARNING :: Ensure that the virtual environment is activated before you begin your installation.
-
-To install AxonDeepSeg, "clone" AxonDeepSeg's repository (you will need to have ``git`` installed on your system)::
-
-    git clone https://github.com/neuropoly/axondeepseg.git
+Once your virtual environment is installed and activated, install the AxonDeepSeg software with the following commands::
 
-Then, in your Terminal, go to the AxonDeepSeg folder and install the AxonDeepSeg package with the following commands::
-
-    cd axondeepseg
```

Comment:
On line 102 below, we should add `model_seg_pns_bf` to the list of model folders that will be re-dowload.